LDM-148.tex

```diff
@@ -158,6 +158,31 @@ \section{Summary Concept of Operations}\label{summary-concept-of-operations}
 computing environments including the Data Access Centers, acting as the spine that
 supports them all.
 
+\subsection{Data Flow}\label{data-flow}
+
+Images (science and calibration) are taken on the mountain and stored in the camera image buffer (sized for a week or more).
+Images are copied by the camera software to the Camera Diagnostic Cluster for rapid analysis.
+Images may reside on that cluster for a short time.
+Images are also transferred to the Observatory Operations Data Service (OODS) for use by the OCS-Controlled Pipeline Service (OCPS), the Commissioning Cluster, and other Observatory systems such as the Active Optics System.
+They are expected to reside in the OODS for at least 30 days.
+At the time of readout, images are sent quickly to the US Data Facility (USDF) for Prompt Processing.
+A copy of the raw images is kept in the Base Facility in Chile as part of the Data Backbone, and an archival copy is also maintained in the Data Backbone at the US Data Facility.
+After an agreed delay period, the raw images are made available to the data rights holders in both the US and Chilean Data Access Centers (DACs) via the LSST Science Platform.
+During the 24 hours after readout, raw images are sent to the Satellite Data Facility at CC-IN2P3 both as a disaster recovery backup and to facilitate the Data Release Processing there at a later date.
```

Comment:
Suggest: "During the 24 hours after readout, raw images are sent to the French Data Facility (FrDF) at CC-IN2P3 ..."

Reply:
"... at a later date. Processing may also be done at other international data facilities."